openfisca_web_api/controllers/__init__.py

```diff
@@ -29,23 +29,25 @@ def index(req):
 
 
 def make_router():
+    routePrefix = '^/api/{}/'.format(environment.country_package_version)
+
     """Return a WSGI application that searches requests to controllers """
     global router
     routings = [
         ('GET', '^/$', index),
         ('GET', '^/api/?$', index),
-        ('POST', '^/api/1/calculate/?$', calculate.api1_calculate),
-        ('GET', '^/api/1/entities/?$', entities.api1_entities),
-        ('GET', '^/api/1/field/?$', field.api1_field),
-        ('GET', '^/api/1/fields/?$', fields.api1_fields),
-        ('GET', '^/api/1/formula/(?P<name>[^/]+)/?$', formula.api1_formula),
-        ('GET', '^/api/2/formula/(?:(?P<period>[A-Za-z0-9:-]*)/)?(?P<names>[A-Za-z0-9_+-]+)/?$', formula.api2_formula),
-        ('GET', '^/api/1/graph/?$', graph.api1_graph),
-        ('GET', '^/api/1/parameters/?$', parameters.api1_parameters),
-        ('GET', '^/api/1/reforms/?$', reforms.api1_reforms),
-        ('POST', '^/api/1/simulate/?$', simulate.api1_simulate),
-        ('GET', '^/api/1/swagger$', swagger.api1_swagger),
-        ('GET', '^/api/1/variables/?$', variables.api1_variables),
+        ('POST', routePrefix + 'calculate/?$', calculate.api1_calculate),
+        ('GET', routePrefix + 'entities/?$', entities.api1_entities),
+        ('GET', routePrefix + 'field/?$', field.api1_field),
+        ('GET', routePrefix + 'fields/?$', fields.api1_fields),
+        ('GET', routePrefix + 'formula/2/(?:(?P<period>[A-Za-z0-9:-]*)/)?(?P<names>[A-Za-z0-9_+-]+)/?$', formula.api2_formula),
+        ('GET', routePrefix + 'formula/(?P<name>[^/0-9][^/]+)/?$', formula.api1_formula),
+        ('GET', routePrefix + 'graph/?$', graph.api1_graph),
+        ('GET', routePrefix + 'parameters/?$', parameters.api1_parameters),
+        ('GET', routePrefix + 'reforms/?$', reforms.api1_reforms),
+        ('POST', routePrefix + 'simulate/?$', simulate.api1_simulate),
+        ('GET', routePrefix + 'swagger$', swagger.api1_swagger),
+        ('GET', routePrefix + 'variables/?$', variables.api1_variables),
```

Comment:
The idea of the versioning here is :
- I can create a new route `api/{country_package_version}/something/`
- If I want a new version of this route, I create `api/{country_package_version}/something/2/`

I thought having `api/{country_package_version}/something/1/` for every route would be heavy, but no idea if it's a good practise.